lib/opensusebasetest.pm

```diff
@@ -799,6 +803,7 @@ sub handle_grub {
         send_key 'ctrl-x';
     }
     else {
+        $self->tianocore_secureboot_invert_state if (get_var 'DISABLE_SECUREBOOT');
```

Comment:
So if secure boot is for some reason already disabled, this would enable it?

Maybe change to a `tianocore_disable_secureboot` function instead which has a needle match on the disabled setting.

Reply:
That was the initial name I have used during development. But as we are going to use the test module in 2 test suites (sb on, sb off). I will change it back.

The needle located [X] with surrounding black background, but ignored the state inside []. So if the keystroke wouldn't be missed or ignored it should revert the actual state. 

[tianocore-devicemanager-sb-conf-attempt-sb-20200421.png](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/blob/master/tianocore-devicemanager-sb-conf-attempt-sb-20200421.png)